stable/cert-manager-crds/Chart.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v2
+description: A chart that bundles cert-manager CRDs with cert-manager helm chart
+home: https://github.com/mesosphere/kommander
+name: cert-manager-crds
+version: v0.0.1
+appVersion: "0.0.1"
```

Comment:
The `appVersion` should be matching the version of `cert-manager` for which are CRDs installed. In our case `v1.7.1`.